src/ResourceManager/Resources/Commands.Resources/RoleAssignments/GetAzureRoleAssignmentCommand.cs

```diff
@@ -237,6 +237,8 @@ public override void ExecuteCmdlet()
                 ExcludeAssignmentsForDeletedPrincipals = true
             };
 
+            AuthorizationClient.ValidateScope(options.Scope);
```

Comment:
>AuthorizationClient.ValidateScope(options.Scope); [](start = 12, length = 49)

we can validate before creating the options right,if the scope is invalid,there isnt any use of the creation of options